### PR TITLE
Torothin issue #2247 revel in pain

### PR DESCRIPTION
**Pull request by torothin**

### CODE REVIEW

File: src/parser/demonhunter/vengeance/modules/spells/azeritetraits/RevelInPain.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import Events from 'parser/core/Events';
+import AzeritePowerStatistic from 'interface/statistics/AzeritePowerStatistic';
+import { formatNumber, formatPercentage } from 'common/format';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+import React from 'react';
+
+/**
+ * Infernal Armor Azerite Power
```
Comment (emallson) [inline src/parser/demonhunter/vengeance/modules/spells/azeritetraits/RevelInPain.js]: looks like the description is right but title in the comment is wrong?

File: src/parser/demonhunter/vengeance/modules/spells/azeritetraits/RevelInPain.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    
+    // this is how effective the tank was at ensuring the bubble was consumed rather than expired
+    const usedAbsorbPossible = formatPercentage(this.totalAbsorbed/this.totalAbsorbedPossible);
+    
+    return (
+      <AzeritePowerStatistic
+        size="flexible"
+      >
+        <BoringSpellValueText spell={SPELLS.REVEL_IN_PAIN}>
+          <img
```
Comment (emallson) [inline src/parser/demonhunter/vengeance/modules/spells/azeritetraits/RevelInPain.js]: I actually totally skimmed over this. You should use the `<ItemHealing />` element for this.

Comment (torothin) [reply src/parser/demonhunter/vengeance/modules/spells/azeritetraits/RevelInPain.js]: Updated/Fixed

File: src/parser/demonhunter/vengeance/modules/spells/azeritetraits/RevelInPain.js (hunk 1 above)
Comment (torothin) [reply src/parser/demonhunter/vengeance/modules/spells/azeritetraits/RevelInPain.js]: Copy/Paste fails . . . fixed

**Pull request merged**